docs/contribute/community.rst

```diff
@@ -1,51 +1,30 @@
+- Sustained contribution to the project, demonstrated by discussion over RFCs, code reviews and proposals of new features, and other development activities. Familiar with, and being able to take ownership on one or several areas of the project.
+- Quality of contributions: High-quality, readable code contributions indicated by pull requests that can be merged without a substantial code review.  History of creating clean, maintainable code and including good test cases. Informative code reviews to help other contributors that adhere to a good standard.
+- Community involvement: active participation in the discussion forum, promote the projects via tutorials, talks and other outreaches. We encourage committers to collaborate broadly, e.g. do code reviews and discuss designs with community members that they do not interact physically.
 
-- High-quality, readable code contributions indicated by pull requests that can be merged without a substantial code review
-- Good coverage of tests and documentation in the contributions
-- Informative code reviews to help other contributors that adhere to a good standard, spot problems in contributions etc.
-- Active participation in the discussion forum
+The Project Management Committee(PMC) consists group of active committers that moderate the discussion, manage the project release, and proposes new committer/PMC members. Potential candidates are usually proposed via an internal discussion among PMCs, followed by a consensus approval, i.e. least 3 +1 votes, and no vetoes. Any veto must be accompanied by reasoning. PMCs should serve the community by upholds the community practices and guidelines TVM a better community for everyone. PMCs should strive to identify new candidates outside of their own organization.
 
-Reviewer
---------
 
-A reviewer is an individual who actively contributed to the project and is willing to participate in the code review of new contributions. We invite reviewers from active contributors. The reviewer invitation will be sent to the potential reviewer’s email, so please log in to the discussion forum so that we can know which email address we could send an invitation to.
-We actively seek reviews from reviewers. High-quality code reviews prevent technical debt for long-term and are crucial to the success of the project.
-A pull request to the project has to be reviewed by a reviewer in order to be merged.
+Reviewers
+---------
+Reviewers are individuals who actively contributed to the project and is willing to participate in the code review of new contributions. We identify reviewers from active contributors. The committers should explicitly solicit reviews from the reviewer.  High-quality code reviews prevent technical debt for long-term and are crucial to the success of the project. A pull request to the project has to be reviewed by a reviewer in order to be merged.
```

Comment:
-> by at least one reviewer ...

Reply:
-> are willing to participate etc.
-> solicit reviews from reviewers.